sora-android-sdk/src/main/kotlin/jp/shiguredo/sora/sdk/channel/signaling/message/MessageConverter.kt

```diff
@@ -99,6 +99,11 @@ class MessageConverter {
                 msg.simulcastRid = mediaOption.simulcastRid?.toString()
             }
 
+            if (mediaOption.spotlightOption != null && !Sora.usesSpotlightLegacy) {
+                msg.spotlightFocusRid = mediaOption.spotlightOption?.spotlightFocusRid?.toString()
```

Comment:
`mediaOption.spotlightOption?` この ? は必要ですかね? Kotlin コンパイラの気持ちは分からないので、必要ならそのままでいいです。

Reply:
? を外すと以下のエラーが発生したので、このままでいきます

e: /path/to/sora-android-sdk-samples-multi-module/sora-android-sdk/sora-android-sdk/src/main/kotlin/jp/shiguredo/sora/sdk/channel/signaling/message/MessageConverter.kt: (103, 41): Smart cast to 'SoraSpotlightOption' is impossible, because 'mediaOption.spotlightOption' is a mutable property that could have been changed by this time

Reply:
ほーーー、mutable だと smart cast 効かないんですか。
たしかにコンパイラとしてはそれで正しい判断か。
勉強になりました。Thanks!!